docs/sdk/examples/exportpdf.html

```diff
@@ -0,0 +1,178 @@
+			<a href="https://ckeditor.com/cke4/addon/exportpdf">Export to PDF</a> plugin which allows saving the editor content as PDF.
+		</p>
+		<p>
+			It introduces a <span class="button_icon" data-icon="exportpdf" title="Export to PDF">&nbsp;</span><b>new toolbar button</b>.
+			When it's clicked, data from editor is sent to the service maintained by <a href="https://ckeditor.com/ckeditor-cloud-services/">CKEditor Cloud Services</a>.
+			The editor takes care of the received responce and saves it as a file.
+		</p>
+
+		<p class="tip" id="browserSupportInfo" aria-hidden="true">
+			<span class="ie">
+				<strong>Internet Explorer</strong> browser support is limited: the plugin is only available for version 11. If you need to use it in the older one, please <a href="https://ckeditor.com/contact/">contact us</a>.
+			</span>
+		</p>
+
+		<textarea id="editor">
+			<h1 style="text-align:center"><img alt="Bilancino Hotel logo" src="https://ckeditor.com/docs/ckeditor4/latest/examples/assets/image/bilancino-logo.png" style="float:right;height:75px;width:75px;" /><span style="font-family:Georgia,serif">The Flavorful Tuscany Meetup</span></h1>
+
+			<h2 style="text-align:center"><span style="font-family:Georgia,serif"><span style="color:#2980b9">Welcome letter</span></span></h2>
+
+			<p>Dear Guest,</p>
+
+			<p>We are delighted to welcome you to the annual <em>Flavorful Tuscany Meetup</em> and hope you will enjoy the programme as well as your stay at the <a href="https://ckeditor.com">Bilancino Hotel</a>.</p>
+
+			<p>Please find attached the full schedule of the event.</p>
+
+			<blockquote>
+				<p>The annual Flavorful Tuscany meetups are always a culinary discovery. You get the best of Tuscan flavors during an intense one-day stay at one of the top hotels of the region. All the sessions are lead by top chefs passionate about their profession. I would certainly recommend to save the date in your calendar for this one!</p>
+
+				<p>Angelina Calvino, food journalist</p>
+			</blockquote>
+
+			<p>Please arrive at the <a href="https://ckeditor.com">Bilancino Hotel</a> reception desk at least <strong>half an hour earlier</strong> to make sure that the registration process goes as smoothly as possible.</p>
+
+			<p>We look forward to welcoming you to the event.</p>
+
+			<p><img alt="Victoria Valc signature" src="https://ckeditor.com/docs/ckeditor4/latest/examples/assets/image/signature.png" style="height:101px;width:180px" /></p>
+
+			<p><span style="font-size:16px"><strong>Victoria Valc</strong></span></p>
+
+			<p><strong>Event Manager<br />
+			Bilancino Hotel</strong></p>
+
+			<p>&nbsp;</p>
+
+			<div style="page-break-after: always"><span style="display:none">&nbsp;</span></div>
+
+			<p>&nbsp;</p>
+
+			<h2 style="text-align:center"><span style="font-family:Georgia,serif"><span style="color:#2980b9">The Flavorful Tuscany Meetup Schedule</span></span></h2>
+
+			<table border="1" cellspacing="0" style="border-collapse:collapse; width:597px">
```

Comment:
I wonder if we can make the whole table centered? Would it work in generated PDF?

Reply:
Yes, it works :+1: I also centered the content of table.